libmscore/style.cpp

```diff
@@ -361,7 +361,8 @@ static const StyleType styleTypes[] {
       { Sid::showMeasureNumberOne,    "showMeasureNumberOne",    QVariant(false) },
       { Sid::measureNumberInterval,   "measureNumberInterval",   QVariant(5) },
       { Sid::measureNumberSystem,     "measureNumberSystem",     QVariant(true) },
-      { Sid::measureNumberAllStaffs,  "measureNumberAllStaffs",  QVariant(false) },
+      { Sid::measureNumberAllStaves,  "measureNumberAllStaffs",  QVariant(false) }, // need to keep staffs and not staves here not to break score import
```

Comment:
Can't we add another here `{ Sid::measureNumberAllStaves,  "measureNumberAllStaves",  QVariant(false) },` to have a migration path?

Reply:
Hmm, writing "measureNumberAllStaves" into new scores, would break when reading those with older 3.x versions though.
Guess this mighr be something to keep in mind for MuseScore 4 then ;-)

Reply:
In similar cases, haven’t we written *both* tags for compatibility?

Reply:
Hmm, yes, well possible. Not resally sure it is worth the effort though, as it is pure cosmetics and not user visible at all

Reply:
btw, I will update the comment, so that its clearer that it's about backward compatibility,